src/user-guide/running-suites.rst

```diff
@@ -14,29 +14,30 @@ Suite Start-Up
 --------------
 
 There are three ways to start a suite running: *cold start* and *warm start*,
-which start from scratch; and *restart*, which starts from a prior
-suite state checkpoint. The only difference between cold starts and warm starts
-is that warm starts start from a point beyond the suite initial cycle point.
+which start from scratch; and *restart*, which continues from the prior
```

Comment:
Suggestion -
I think that this would be easier to understand in a table
```rst
.. csv-table::
   :header: "Start type", "What happens"

   cold start, "tell me about cold start, in quotes, so that if I bang on
   a bit I can use the next line"
```

You have to read this quite carefully because sentence structure "There are 3 ways to <verb>: A and B <explanation>; and C <more explanation>" is a bit tricky to read IMO.